src/v/coproc/script_context.cc

```diff
@@ -65,29 +69,49 @@ script_context::script_context(
       "valid subscription list");
 }
 
+static void exc_printer(const std::exception_ptr& eptr) {
```

Comment:
small and only nit: this function is called a printer, but it's not only prints exceptions but also filters them, but it's internal so not a big deal

Reply:
Changed